spec/ParseAPI.spec.js

```diff
@@ -1267,6 +1267,35 @@ describe('miscellaneous', function () {
     });
   });
 
+  it('test cloud function query parameters with array of pointers', done => {
+    Parse.Cloud.define('echoParams', req => {
+      return req.params;
+    });
+    const headers = {
+      'Content-Type': 'application/json',
+      'X-Parse-Application-Id': 'test',
+      'X-Parse-Javascript-Key': 'test',
+    };
+    request({
+      method: 'POST',
+      headers: headers,
+      url: 'http://localhost:8378/1/functions/echoParams', //?option=1&other=2
+      qs: {
+        option: 1,
+        other: 2,
+      },
+      body: '{"foo":"bar", "other": 1, "arr": [{ "__type": "Pointer" }]}',
```

Comment:
Is this actually a valid pointer object? It seems to be missing the classname and objectId?

Do we need `foo: bar` and `other: 1` in this test?